src/components/CreditCard.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+
+function CreditCard(props) {
+    const { type, number, expirationMonth, expirationYear, bank, owner, bgColor, color } = props;  
+    const randomNumber = number.slice(-4);
+      return(
+        <div>
+<p>{type}</p>
+<p>{randomNumber}</p>
+<p>{expirationMonth}</p>
+<p>{expirationYear}</p>
+<p>{bank}</p>
+<p>{owner}</p>
+<div style={{ backgroundColor: `${bgColor}`, color: `${color}`}}></div>
```

Comment:
The background color is not displaying because the div is empty. This div should be wrapping all the information in the credit card.